validator/validator-css.protoascii

```diff
@@ -29,6 +29,7 @@ declaration_list {
   declaration: { name: "animation-name" }
   declaration: { name: "animation-play-state" }
   declaration: { name: "animation-timing-function" }
+  declaration: { name: "aspect-ratio" }
```

Comment:
For the email spec, this only modifies the non strict mode spec. To allow it for strict mode spec, we need to add this to `EMAIL_SPECIFIC_DECLARATIONS`. The strict mode spec doesn't use `BASIC_DECLARATIONS`.

Reply:
I added `aspect-ratio` to `EMAIL_SPECIFIC_DECLARATIONS` as well.